test/integration/suites/server-cmds/01-start-server

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
Since we may have integration tests with a lot of different systems, perhaps "server-cmds" is a little vague and we me have some collisions with other "server" software in the future. I think that something like "spire-server-cli" would be a better choice.